### PR TITLE
add section on scientific software wrapper to package categories

### PR DESCRIPTION
**Pull request by sckott**
via slack discussion

### CODE REVIEW

**Comment by noamross** [comment]
I made some small changes, splitting the platform-specific component out to the packaging guide. This is big enough that I'd like feedback (or just 👍 / 👎) from all @ropensci/editors.

@richelbilderbeek, your pre-submission prompted quite a bit of discussion, and we realized we needed to include a new category to capture some types of packages we've been handling.  Take a look and feel free to comment.

File: pkg_building.Rmd [hunk 1]
```diff
@@ -26,6 +26,11 @@ We strongly recommend that package developers read Hadley Wickham's concise but
 We recommend you to use the [`codemetar` package](https://github.com/ropensci/codemetar) for creating and updating a JSON [CodeMeta](https://codemeta.github.io/) metadata file for your package via `codemetar::write_codemeta()`. It will automatically include all useful information, including [GitHub topics](#grooming). CodeMeta uses [Schema.org terms](https://schema.org/) so as it gains popularity the JSON metadata of your package might be used by third-party services, maybe even search engines. 
 
 
+## Platforms
+
+* Packages should run on all major platforms (Windows, macOS, Linux).  Exceptions may be granted packages that interact with system-specific functions, or wrappers for utilties that only operate on limited platforms, but authors should make every effort for cross-platform compatibility, including system-specific compilation, or containerization of external utilities.
```
Comment (maelle) [inline pkg_building.Rmd]:
```suggestion
* Packages should run on all major platforms (Windows, macOS, Linux).  Exceptions may be granted packages that interact with system-specific functions, or wrappers for utilities that only operate on limited platforms, but authors should make every effort for cross-platform compatibility, including system-specific compilation, or containerization of external utilities.
```

**Comment by maelle** [comment]
Thanks @sckott and @noamross, it looks great, only added a typo fix suggestion.

**Comment by richelbilderbeek** [comment]
I am pleasantly surprised that there was a bigger discussion on this: rOpenSci must be a committed community! I am happy that a decision is proposed (in this case: a package to call a single-OS tool is acceptable) and I hope this will encourage authors of such a tool to broaden their scope.

Keep up the good job :+1:

**Comment by karthik** [comment]
I like these changes @sckott and @noamross 👍 
LGTM

**Comment by noamross** [comment]
Re-tagging @ropensci/editors as I realized Brooke and Melina were not previous part of that group

**Comment by maelle** [comment]
@sckott could you please also update the NEWS appendix? Thank you!

**Comment by sckott** [comment]
yes

**Comment by sckott** [comment]
does the news change look okay?

**Pull request merged**